package.json

```diff
@@ -57,7 +61,7 @@
   "devDependencies": {
     "@types/eslint": "^8.56.2",
     "@types/lodash": "^4.17.0",
-    "@types/node": "^20.11.20",
+    "@types/node": "^20.12.7",
```

Comment:
Are we using the node 20.12.7 version?

Reply:
We are using node 20 lts (iron)
I aligned the corresponding .nvm file to always point to the latest lts version and updated package semver to fetch the latest version, please note that [types and node are now always aligned](https://github.com/definitelytyped/definitelytyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library)